contrib/coreos/user-data.example

```diff
@@ -133,3 +134,14 @@ write_files:
       TOOLBOX_DOCKER_IMAGE=ubuntu-debootstrap
       TOOLBOX_DOCKER_TAG=14.04
       TOOLBOX_USER=root
+  - path: /etc/environment_proxy
```

Comment:
I'd rather not overwrite /etc/environment in our logic, as we would potentially be clobbering changes made upstream unless we account for them ourselves. I don't think `/etc/deis/environment` is appropriate either, as the changes aren't specific to Deis (Docker also reads the file). So, I'm not in love with `/etc/environment/proxy`, but not sure if there's a better candidate.